source/configure/plugins-configuration-settings.rst

```diff
@@ -406,6 +406,38 @@ The URL to a running `rtcd <https://github.com/mattermost/rtcd>`__ service insta
 
 This is an optional field. Changing this setting requires a plugin restart to take effect.
 
+Enable call recordings (beta)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. include:: ../_static/badges/ent-selfhosted-only.rst
+  :start-after: :nosearch:
+
+**True**: Allow call hosts to record meeting video and audio on the cloud. Recordings include the entire call window view along with participants' audio track and any shared screen video.
+
+**False**: Call recording functionality is not available to hosts.
+
+Changing this setting requires a plugin restart to take effect.
+
+Job service URL
+~~~~~~~~~~~~~~~
+
+.. include:: ../_static/badges/ent-selfhosted-only.rst
+  :start-after: :nosearch:
+
+The URL to the instance where call recordings will be stored.
```

Comment:
Is there a default for this config setting?

Reply:
I can't find one in the file. @streamer45, is there a default value for this?

Reply:
There's no default as it's a required setting that should be provided by the system admin pointing to a running service.